File: Back/src/main/java/conexion/mysql.java
package conexion;

import java.sql.DriverManager;
import java.sql.SQLException;
import org.springframework.boot.autoconfigure.amqp.RabbitProperties.Cache.Connection;

public class mysql {


    public static Connection getConexion(){
        String conexionUrl = "jdbc:sqlserver://usbdwuhbxcy3cjxt:EvhPH1RIMjnxdiG0S8vP@bjtuc0nkttom2clvwxqt-mysql.services.clever-cloud.com:3306/bjtuc0nkttom2clvwxqt;"
        +"database=bjtuc0nkttom2clvwxqt;"
        +"user=usbdwuhbxcy3cjxt;"
        +"password=EvhPH1RIMjnxdiG0S8vP;"
        +"loginTimeout=30;";

        try{
            Connection con = DriverManager.getConnection(conexionUrl);
            return con;
        }catch(SQLException ex){
            System.out.println(ex.toString());
            return null;
        }
    }

}

File: Front/fronted/src/mysql.js
const mysql = require ('mysql')
const connection = mysql.createConnection({
    host:'bjtuc0nkttom2clvwxqt-mysql.services.clever-cloud.com',
    user:'usbdwuhbxcy3cjxt',
    password:'EvhPH1RIMjnxdiG0S8vP',
    database:'bjtuc0nkttom2clvwxqt'
})

connection.connect((err)=>{
    if (err) throw err
    console.log('Conexion establecida exitosamente!')
})

//AGREGAR INFORMACION A LAS TABLAS DE LA BASE DE DATOS+++++
const insertar = "INSERT INTO alumno VALUES ('2','Karen','015487360')"
connection.query(insertar,(err, rows)=>{
    if(err) throw err
    console.log('Los datos insertados con exito')
})

//CONSULTAR LA BASE DE DATOS POR TABLAS Y MOSTRAR LA INFO
connection.query('Select * from alumno', (err, rows)=>{
    if(err) throw err
    console.log('Los datos solicitados son:')
    console.log(rows)
})

connection.end()